CI looks to be failing;

```
13:16:18 --- FAIL: Test (3265.54s)
13:16:18 panic: DockerSwarmSuite.TestSwarmNetworkPlugin test timed out after 5m0s [recovered]
13:16:18 	panic: DockerSwarmSuite.TestSwarmNetworkPlugin test timed out after 5m0s
13:16:18 
13:16:18 goroutine 8 [running]:
13:16:18 panic(0x12257a0, 0xc42063e010)
13:16:18 	/usr/local/go/src/runtime/panic.go:500 +0x1a1
13:16:18 testing.tRunner.func1(0xc42007a840)
13:16:18 	/usr/local/go/src/testing/testing.go:579 +0x25d
13:16:18 panic(0x12257a0, 0xc42063e010)
...
```

--- a/daemon/network.go
+++ b/daemon/network.go
@@ -407,6 +407,13 @@ func (daemon *Daemon) GetNetworkDriverList() []string {
 	}
 
 	pluginList := daemon.netController.BuiltinDrivers()
+
+	managedPlugins := daemon.PluginStore.GetAllManagedPluginsByCap(driverapi.NetworkPluginEndpointType)
+
+	for _, plugin := range managedPlugins {
+		pluginList = append(pluginList, plugin.Name())
+	}
+
 	pluginMap := make(map[string]bool)
 	for _, plugin := range pluginList {
 		pluginMap[plugin] = true

